internal/legacy/options/options_gssapi.go

```diff
@@ -13,11 +13,11 @@ func init() {
 }
 
 func registerGSSAPIOptions(self *ToolOptions) error {
-	_, err := self.parser.AddGroup("kerberos options", "", self.Kerberos)
+	_, err := self.parser.AddGroup("kerberos options", "", Kerberos)
```

Comment:
Why is this changing? Here and below?

Reply:
Ah, the IDE did the trick when I moved the file to a new folder. I fixed a few but missed these ones. Thanks for catching this!